doc/modules/preprocessing.rst

```diff
@@ -589,60 +589,6 @@ columns for this feature will be all zeros
 See :ref:`dict_feature_extraction` for categorical features that are represented
 as a dict, not as scalars.
 
-.. _imputation:
```

Comment:
Please keep this and leave a note pointing to the new document, otherwise some links will be broken.
E.g., https://19893-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.preprocessing.Imputer.html#sklearn.preprocessing.Imputer
(or you may also correct existing links)